Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.py

```diff
@@ -1274,8 +1350,267 @@ def test_function(client: MsGraphClient, args, has_access_to_context=False):
                      f'Please check authentication related parameters. [{response.status_code}]')
 
 
+def get_message_user(client, message_user):
+    is_email = re.search(EMAIL_REGEX, message_user)
+    if is_email:
+        user_result = (client.get_user_id(message_user)).get('value')
```

Comment:
does al response from client.get_user_id contain value ? can it return with no value and throw exception ?

Reply:
if there's no user, the value is returned as an empty list